sample_lp_sever_notebooks/mixed-integer-linear-programming.ipynb

```
@@ -47,8 +47,47 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Check for GPU\n",
-    "!nvidia-smi\n"
+    "import subprocess\n",
+    "from IPython.display import display, HTML\n",
```

Ruff (0.14.0)

Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)